_config.yml

```diff
@@ -109,6 +109,11 @@ authors:
     display_name: Honza Javorek
     email: "honza@apiary.io"
     twitter: "@honzajavorek"
+  daria:git log
```

Comment:
Oh, `git log` is here. I think that was by accident. @dariagrudzien can you please remove.